Water Drink Water/src/api/Endpoints/Preferences/Get.cs

```diff
@@ -0,0 +1,35 @@
+﻿using System.Security.Claims;
+using FastEndpoints;
+using Microsoft.AspNetCore.Http.HttpResults;
+using TbdFriends.WaterDrinkWater.Application.Services;
+
+namespace TbdFriends.WaterDrinkWater.Api.Endpoints.Preferences;
+
+public class Get(ConsumptionService service) : EndpointWithoutRequest<Results<Ok<PreferencesVm>, BadRequest>>
+{
+    public override void Configure()
+    {
+        Get("api/preferences");
+    }
+
+    public override Task<Results<Ok<PreferencesVm>, BadRequest>> ExecuteAsync(CancellationToken ct)
+    {
+        if (int.TryParse(
+                User.Claims.SingleOrDefault(c => c.Type == ClaimTypes.NameIdentifier)?.Value.Split("|")[1],
+                out var userId))
+        {
+            var preferences = service.GetPreferences(userId);
+            return Task.FromResult<Results<Ok<PreferencesVm>, BadRequest>>(TypedResults.Ok(new PreferencesVm
+            {
+                TargetFluidOunces = preferences
+            }));
+        }
+
+        return Task.FromResult<Results<Ok<PreferencesVm>, BadRequest>>(TypedResults.BadRequest());
+    }
+}
+
+public class PreferencesVm
```

Comment:
At this point it can be inside the class, we can refactor it to live there, but maybe that is not the best design in the long run, we might move the models to a separate shared class so the client and the api both talk of the same models